### PR TITLE
Verify

### LINKED ISSUE #25: add a register auth page
something like: '/auth/register?token={token}'

### PR DESCRIPTION
**Pull request by PurpurTentakel97**
close #25 

- dashboard tabs turned from buttons to styled links
- some console.log() removed
- add verify page

### CODE REVIEW

File: src/routes/verify-email-address/+page.svelte [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+	<Header menu={Menu.header} />
+	<div class="verify-email-content-wrapper">
+		<HeadlinePage>E-Mail Verifizierung</HeadlinePage>
+		{#if data.success === undefined}
+			<div class="verify-email-text-wrapper">
+				<TextLine>👻 Wir versuchen gerade deine E-Mail zu verifizieren. 👻</TextLine>
+				<TextLine>Offenbar dauert das alles ein bisschen länger als wir erwartet haben.</TextLine>
+				<TextLine>Bitte hab etwas Geduld</TextLine>
+			</div>
+			<TextLine>Falls sich hier länger nix tun sollte melde dich gerne bei uns.</TextLine>
```
Comment (mgerhold) [inline src/routes/verify-email-address/+page.svelte:24]:
```suggestion
			<div class="verify-email-text-wrapper">
				<TextLine>👻 Wir versuchen gerade, deine E-Mail zu verifizieren... 👻</TextLine>
				<TextLine>Offenbar dauert das alles ein bisschen länger, als wir erwartet haben.</TextLine>
				<TextLine>Bitte hab etwas Geduld</TextLine>
			</div>
			<TextLine>Falls sich hier länger nix tun sollte, melde dich gerne bei uns.</TextLine>
```

File: src/routes/verify-email-address/+page.svelte [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+			<StyledLink
+				text={MenuItem.imprint.name}
+				href={MenuItem.imprint.url}
+				title={MenuItem.imprint.description}
+				newTab={false}
+			/>
+		{:else if data.success}
+			<div class="verify-email-text-wrapper">
+				<TextLine>Deine E-Mail wurde erfolgreich verifiziert.</TextLine>
+				<TextLine>Du kannst dich ab jetzt anmelden.</TextLine>
```
Comment (mgerhold) [inline src/routes/verify-email-address/+page.svelte:34]: Just personal preference...I like this a bit more:
```suggestion
				<TextLine>Du kannst dich jetzt anmelden.</TextLine>
```

File: src/routes/verify-email-address/+page.svelte [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+			</div>
+			<StyledLink
+				text={MenuItem.login.name}
+				href={MenuItem.login.url}
+				title={MenuItem.login.description}
+				newTab={false}
+			></StyledLink>
+		{:else}
+			<div class="verify-email-text-wrapper">
+				<TextLine>🫨 Verifizierung leider nicht erfolgreich. 🫨</TextLine>
```
Comment (mgerhold) [inline src/routes/verify-email-address/+page.svelte:44]: I love those emojis 😂

File: src/routes/verify-email-address/+page.ts [hunk 4]
```diff
@@ -0,0 +1,34 @@
+import { apiUrl } from "helper/links";
+import { checkAndParseGlobalsAsync } from "helper/parseJson";
+import type { LoadVerifyEmail } from "types/loadTypes";
+import type { Globals } from "types/provideTypes";
+
+export const ssr = false;
```
Comment (mgerhold) [inline src/routes/verify-email-address/+page.ts:6]: Maybe we should add a comment here to explain why we disabled server-side rendering? I can imagine that we'll forget that later.

Comment (PurpurTentakel97) [reply src/routes/verify-email-address/+page.ts]: Given the fact that I already forgot about this.
I added a comment. 
😂

**Review by mgerhold — APPROVED** [review]

**Pull request merged**